### PR TITLE
Metamath: init at 0.167

### PR DESCRIPTION
**Pull request by Taneb**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Taneb** [comment]
This points to my own repository because there is no official repository for metamath, there's a download location but it gets updated in place with no permanent link. See discussion at https://groups.google.com/forum/#!topic/metamath/N4WEWQQVUfY

File: pkgs/development/interpreters/metamath/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "metamath-${version}";
+  version = "0.167";
+
+  buildInputs = [ autoreconfHook ];
+
+  src = fetchFromGitHub {
```
Comment (rasendubi) [inline pkgs/development/interpreters/metamath/default.nix]: Please copy the comment from GitHub on why we use a mirror.

File: pkgs/development/interpreters/metamath/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  meta = with stdenv.lib; {
+    description = "Interpreter for the metamath proof language";
+    longDescription = ''
+      The metamath program is an ASCII-based ANSI C program with a command-line
+      interface. It was used (along with mmj2) to build and verify the proofs
+      in the Metamath Proof Explorer, and it generated its web pages. The *.mm
+      ASCII databases (set.mm and others) are also included in this derivation.
+      '';
+    homepage = "https://us.metamath.org";
+    downloadPage = "http://us.metamath.org/downloads/metamath.tar.gz";
```
Comment (rasendubi) [inline pkgs/development/interpreters/metamath/default.nix]: It doesn't look like download "page."
```suggestion
    downloadPage = "http://us.metamath.org/#downloads";
```

File: pkgs/development/interpreters/metamath/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  };
+
+  meta = with stdenv.lib; {
+    description = "Interpreter for the metamath proof language";
+    longDescription = ''
+      The metamath program is an ASCII-based ANSI C program with a command-line
+      interface. It was used (along with mmj2) to build and verify the proofs
+      in the Metamath Proof Explorer, and it generated its web pages. The *.mm
+      ASCII databases (set.mm and others) are also included in this derivation.
+      '';
```
Comment (rasendubi) [inline pkgs/development/interpreters/metamath/default.nix]: De-indent.
```suggestion
    '';
```

**Comment by rasendubi** [comment]
Could you also squash commits? (`Add Taneb as maintainer` doesn't look like a good commit name in isolation.)

@GrahamcOfBorg build metamath

**Comment by Taneb** [comment]
Thanks for the feedback! I've made the suggested changes and squashed

**Review by rasendubi — APPROVED** [review]

**Comment by rasendubi** [comment]
@GrahamcOfBorg build metamath

**Pull request merged**